components/automate-ui/src/app/services/projects-filter/projects-filter.reducer.ts

```diff
@@ -36,22 +52,25 @@ export function projectsFilterReducer(
   switch (action.type) {
 
     case ProjectsFilterActionTypes.LOAD_OPTIONS: {
-      return set('optionsLoadingStatus', LoadingStatus.loading, state);
+      return set('optionsLoadingStatus', EntityStatus.loading, state);
     }
 
     case ProjectsFilterActionTypes.LOAD_OPTIONS_SUCCESS: {
+      const merged = mergeOptions(action.payload.fetched, action.payload.restored);
+      const sortedOptions = sortOptions(merged);
```

Comment:
random nit: would go with `mergedOptions/sortedOptions` or `merged/sorted` for these var names.